test/unit/workflowapi/test_workflowapi_app.py

```diff
@@ -562,7 +562,7 @@
         'Media': {
             'Video': {
                 'S3Bucket': 'testBucket',
-                'S3Key': 'private/assets/abcd-1234-efgh/input/testFile.mp4'
+                'S3Key': 'testFile.mp4'
```

Comment:
Now that the dataplane api is no longer copying input media files to /private/assets/, this test does not need to include that folder in the input validation.